pkg/provider/cloud/kubevirt/network_policy.go

```diff
@@ -103,3 +117,19 @@ func reconcileClusterImagesNetworkPolicy(ctx context.Context, cluster *kubermati
 	}
 	return nil
 }
+
+// infrKubeVirtClusterPodCidr returns the infra KubeVirt cluster pod CIDR.
+func infrKubeVirtClusterPodCidr(ctx context.Context, client ctrlruntimeclient.Client) (string, error) {
+	nodes := corev1.NodeList{}
+	if err := client.List(ctx, &nodes); err != nil {
+		return "", fmt.Errorf("Failed to list nodes: %w", err)
+	}
+	cidr := ""
+	for _, node := range nodes.Items {
+		if len(node.Spec.PodCIDRs) > 0 {
+			cidr = node.Spec.PodCIDRs[0]
+		}
+	}
+
```

Comment:
We have to collect a list of CIDRs and add them to network policy. Each node can have a different CIDR.
So that would be to take everything from `node.Spec.PodCIDRs`  and it should be done for each node, as a result you would have a set of CIDRs that should be attached to network policy.

Reply:
Right, done